src/libcore/mem/maybe_uninit.rs

```diff
@@ -33,10 +30,7 @@ use crate::mem::ManuallyDrop;
 ///
 /// ```rust,no_run
 /// # #![allow(invalid_value)]
-/// use std::mem::{self, MaybeUninit};
-///
-/// let b: bool = unsafe { mem::uninitialized() }; // undefined behavior!
```

Comment:
When removing this example here, please add it in the `mem::uninitialized` docs (as an explicitly incorrect usage) so that the example does not get entirely lost.